samples/attested_tls/non_enc_client/verify_callback.cpp

```diff
@@ -40,13 +41,27 @@ oe_result_t enclave_identity_verifier(oe_identity_t* identity, void* arg)
         TLS_CLIENT "identity.security_version = %d\n",
         identity->security_version);
 
-    printf(TLS_CLIENT "identity->unique_id : ");
+    // the unique ID for the enclave, for SGX enclaves, this is the MRENCLAVE
+    // value
+    printf(TLS_CLIENT "Validating identity->unique_id(MRENCLAVE) :\n");
     for (int i = 0; i < OE_UNIQUE_ID_SIZE; i++)
+    {
         printf("0x%0x ", (uint8_t)identity->unique_id[i]);
+        if (SERVER_ENCLAVE_MRENCLAVE[i] != (uint8_t)identity->unique_id[i])
+        {
```

Comment:
will you set an error here?

Reply:
the default result is OE_VERIFY_FAILED, so that should cover it.